### PR TITLE
Adding "pop"

### PR DESCRIPTION
**Pull request by MisterJames**
- added microphone.png image
- moved podcast detection up in structure
- added floating div
- removed height restrictions on home page
- removed ellipsis class from css
- added podcast-icon class

Addresses #124

### CODE REVIEW

**Comment by MisterJames** [comment]
image

Let it flow, let it flow, let it flow.

Also: microphones because podcasts.

**Comment by kbaley** [comment]
I think letting it flow leads to too much whitespace on the right under Speaking. I like the idea of letting it flow in the full post listing but it should be capped here on the home page.

File: themes/westerndevs/layout/_widget/home_page/recent_posts.ejs [hunk 1]
```diff
@@ -18,10 +19,14 @@
           <% if (post.author && post.author.name ) { %>
           <B><%= post.author.name %></B><BR>
           <% } %>
-          <div class="ellipsis">
+          <div>
+	  <% if(cat) { %>
+			<div class="pull-right"><img src="images/microphone.png" alt="Podcast" title="Podcast" class="podcast-icon" /></div>
```
Comment (kbaley) [inline themes/westerndevs/layout/_widget/home_page/recent_posts.ejs]: How does our existing icon_podcasts.png image look here?

If it looks funny, consider using font-awesome instead of an image:

`<i class="fa-microphone" />`

**Comment by MisterJames** [comment]
@kbaley Why is whitespace bad there? I don't think anyone is going to be printing it out and trying to get it all on one page. :P  One of my pet peeves is when things are cut off in mid sentence without knowing what the rest of the paragraph was going

I'll have a look at the other two options for the podcast. I looked at loaded resources and we didn't seem to have FA loaded in the browser when the page rendered (I suppose it's bundled?).

**Comment by kbaley** [comment]
It's more of a gestalt thing. I think it looks less sloppy if the posts and the right sidebar align better, even if it's not 100%. In any case, we all have our pet peeves. Mine is "summaries" that are seven lines long. This could be solved if we had better summaries for each post.

Note that our previous site also cut the summaries off after a certain number of characters. It's how Jekyll handles excerpts. If one isn't provided, it grabs the first 200 or so characters from the post. Hexo doesn't have the same functionality. Instead, you need to add a `<!--more-->` in the post and everything above that counts as the excerpt. So I just added that to all posts that didn't explicitly set an excerpt just so we had one for every post.

To me the home page is an introduction to what we're all about and my feeling is that the overall look and feel should be taken into consideration. That's why we include speaking engagements, what we've done, and who we are. And it's why I suggested letting the lengthier summaries flow in the posts page rather than the home page.

So to sum up, I think the best way to handle this: let the summaries flow but make sure all future posts (and the five most recent ones) have reasonably-sized "excerpts".

**Comment by kbaley** [comment]
Here's how it looks with our podcast icon:

image

**Comment by kbaley** [comment]
And with a slightly smaller microphone (from FA) in the same colour as our icons.

image

**Comment by MisterJames** [comment]
I think I like the FA one.

As far as shape goes, maybe we're not too far off from each other. I'm thinking - from the gestalt perspective - that the hard cut offs are what bugs me. Agreed that decent summaries are needed, but a stringent grid with everything at equal spacing (when it's sooooooo apparent that there is more content) looks so 2003.

**Comment by kbaley** [comment]
And finally, with a volume-up icon (essentially, our podcast icon inverted):

image

**Comment by MisterJames** [comment]
I like the inverted ones best, either the volume-up or the mic. I prefer those over the big circle

**Comment by kbaley** [comment]
Agreed. Our podcast icon is too much. I don't have a preference which icon we use either. They both look good and make it clear there's sound involved.

**Comment by kbaley** [comment]
WRT letting it flow, let's do that then and make a point of setting appropriate excerpts on any posts that have too large of one now. Go ahead and switch the microphone to FA and merge it.

**Comment by MisterJames** [comment]
@kbaley should be good to go. Thanks for the feedback.

**Comment by kbaley** [comment]
Delete microphone.png and then you can merge
